packages/plugin-eslint/src/lib/nx/find-project-without-deps.ts

```diff
@@ -0,0 +1,39 @@
+import type { ESLintTarget } from '../config';
+import { nxProjectsToConfig } from './projects-to-config';
+
+/**
+ * Accepts a target Nx projects, converts lint configurations to Code PushUp ESLint plugin parameters.
+ *
+ * Use when you wish to include a targeted subset of your Nx monorepo in your Code PushUp project.
+ * If you prefer to include all Nx projects, refer to {@link eslintConfigFromAllNxProjects} instead,
+ * if you'd like to auto include all dependencies of the provided target project use {@link eslintConfigFromNxProjectAndDeps} instead.
+ *
+ * @example
+ * import eslintPlugin, {
+ *   eslintConfigFromNxProject,
+ * } from '@code-pushup/eslint-plugin';
+ *
+ * const projectName = 'backoffice'; // <-- name from project.json
+ *
+ * export default {
+ *   plugins: [
+ *     await eslintPlugin(
+ *       await eslintConfigFromNxProject(projectName)
+ *     )
+ *   ]
+ * }
+ *
+ * @param projectName Nx project serving as main entry point
+ * @returns ESLint config and patterns, intended to be passed to {@link eslintPlugin}
+ */
+export async function eslintConfigFromNxProject(
+  projectName: string,
+): Promise<ESLintTarget[]> {
+  const { createProjectGraphAsync } = await import('@nx/devkit');
+  const projectGraph = await createProjectGraphAsync({ exitOnError: false });
+
+  return nxProjectsToConfig(
+    projectGraph,
+    project => !!project.name && project.name === projectName,
+  );
+}
```

Comment:
Since there can only be one matching project, this could return a single lint target instead of array. Would also be useful to inform the user if the Nx project name is wrong.

```suggestion
export async function eslintConfigFromNxProject(
  projectName: string,
): Promise<ESLintTarget> {
  const { createProjectGraphAsync } = await import('@nx/devkit');
  const projectGraph = await createProjectGraphAsync({ exitOnError: false });

  const [project] = nxProjectsToConfig(
    projectGraph,
    project => !!project.name && project.name === projectName,
  );
  
  if (!project) {
    throw new Error(`Couldn't find Nx project named "${projectName}"`);
  }
  return project;
}
```